_component_design/Eyebrow.md

```diff
@@ -0,0 +1,430 @@
+---
+title: Eyebrow
+parent: Navigation
+layout: component
+category: Components
+usage: >
+  The eyebrow is the top menu bar across all breakpoints within the application frame of a Helix interface. The eyebrow provides access to platform-level services and aggregate information across a user's accounts at Rackspace.
+preview-image: preview-images/eyebrow.svg
+status: stable
+resource: true
+helix-ui-css: true
+helix-ui-javascript: true
+last-modified: 2017-08-30
+---
+
+<!-- {% include toc.html %} commented out due to the size of the doc -->
```

Comment:
So are we starting to skip TOCs?

Reply:
The ToC on this page was really unruly so I omitted it on this doc. This is what prompted the discussion around updating the ToC style / structure in slack last week.

![image](https://user-images.githubusercontent.com/586682/34118036-34e366da-e3e3-11e7-8200-9fa303b63563.png)